examples/src/main/python/ml/tokenizer_example.py

```diff
@@ -18,11 +18,11 @@
 from __future__ import print_function
 
 # $example on$
-from pyspark.ml.feature import Tokenizer, RegexTokenizer
-from pyspark.sql.functions import col, udf
-from pyspark.sql.types import IntegerType
+from pyspark.ml.feature import RegexTokenizer, Tokenizer
 # $example off$
 from pyspark.sql import SparkSession
+from pyspark.sql.functions import col, udf
```

Comment:
These probably should be be included in the example (they were before), so add an # $example on$ / # $example off$ around them.